cms/package.json

```diff
@@ -14,6 +14,7 @@
     "@strapi/plugin-i18n": "4.13.1",
     "@strapi/plugin-users-permissions": "4.13.1",
     "@strapi/strapi": "4.13.1",
+    "lodash": "^4.17.21",
```

Comment:
better use https://www.npmjs.com/package/lodash-es

Reply:
this should actually NOT be here, but inside the plugin itself